xml
CopyEdit
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SitrepForm</class>
 <widget class="QDialog" name="SitrepForm">
  <property name="windowTitle">
   <string>SITREP</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabHeader">
      <attribute name="title"><string>Заголовок</string></attribute>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Категория сообщения:</string></property></widget></item>
       <item row="0" column="1"><widget class="QComboBox" name="comboCategory"><item><property name="text"><string>Бедствие</string></property></item><item><property name="text"><string>Тренировка</string></property></item></widget></item>
       <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Дата и время (UTC):</string></property></widget></item>
       <item row="1" column="1"><widget class="QDateTimeEdit" name="dateTimeUtc"/></item>
       <item row="2" column="0"><widget class="QLabel"><property name="text"><string>От кого:</string></property></widget></item>
       <item row="2" column="1"><widget class="QLineEdit" name="editFrom"/></item>
       <item row="3" column="0"><widget class="QLabel"><property name="text"><string>Кому:</string></property></widget></item>
       <item row="3" column="1"><widget class="QLineEdit" name="editTo"/></item>
       <item row="4" column="0"><widget class="QLabel"><property name="text"><string>Дополнительно:</string></property></widget></item>
       <item row="4" column="1"><widget class="QTextEdit" name="textAdditional"/></item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMainInfo">
      <attribute name="title"><string>Основная информация</string></attribute>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Объект аварии:</string></property></widget></item>
       <item row="0" column="1"><widget class="QLineEdit" name="editObject"/></item>
       <item row="1" column="0"><widget class="QLabel"><property name="text"><string>Местоположение:</string></property></widget></item>
       <item row="1" column="1"><widget class="QLineEdit" name="editLocation"/></item>
       <item row="2" column="0"><widget class="QLabel"><property name="text"><string>Координаты (широта):</string></property></widget></item>
       <item row="2" column="1"><widget class="QLineEdit" name="editLat"/></item>
       <item row="3" column="0"><widget class="QLabel"><property name="text"><string>Координаты (долгота):</string></property></widget></item>
       <item row="3" column="1"><widget class="QLineEdit" name="editLon"/></item>
       <item row="4" column="0"><widget class="QLabel"><property name="text"><string>Ситуация:</string></property></widget></item>
       <item row="4" column="1"><widget class="QTextEdit" name="textSituation"/></item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInfoG">
      <attribute name="title"><string>Доп. информация G</string></attribute>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Размер (Д×Ш×Осадка):</string></property></widget></item>
       <item row="0" column="1"><widget class="QLineEdit" name="editSize"/></item>
       <item row="1" column="0"><widget class="QLabel"><property name="text"><string>MMSI/IMO:</string></property></widget></item>
       <item row="1" column="1"><widget class="QLineEdit" name="editMMSI"/></item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInfoHJ">
      <attribute name="title"><string>Доп. информация H,J</string></attribute>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Погода (ветер, течение):</string></property></widget></item>
       <item row="0" column="1"><widget class="QTextEdit" name="textWeather"/></item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInfoKLMN">
      <attribute name="title"><string>K,L,M,N</string></attribute>
      <layout class="QFormLayout">
       <item row="0" column="0"><widget class="QLabel"><property name="text"><string>Район поиска (K):</string></property></widget></item>
       <item row="0" column="1"><widget class="QTextEdit" name="textSearchArea"/></item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSend">
     <property name="text"><string>Отправить</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
жду
Публикую далее. Следующие формы — авиация, планшет, план поиска.
